src/components/sections/FavouriteSchools/FavouriteSchool.tsx

```diff
@@ -51,36 +41,37 @@ const FavouriteSchool: FC<FavouriteSchoolProps> = ({ schoolID, toggleFavourite }
             </RemoveFavButton>
           </div>
         </div>
-
-        <div className="grid">
-          <div>
-            <h5>Tegoroczne profile</h5>
-            <ul>
-              {profiles.map((profile: any) => (
-                <li key={nanoid()}>
-                  <span>{profile.extendedSubjects.join(' - ')}</span>
-                </li>
-              ))}
-            </ul>
-          </div>
-          <div>
-            <h5>Zeszłoroczne progi punktowe</h5>
-            {classes && classes.length > 0 ? (
-              <ul>
-                {classes.map((c: any) => (
-                  <li key={nanoid()}>
-                    <span>{c.subjects.map((s: any) => s.name).join('-')}</span>
-                    <span className="points">
-                      {c.stats && c.stats[0].points_min > 0 && <>{c.stats[0].points_min} pkt</>}
-                    </span>
-                  </li>
-                ))}
-              </ul>
-            ) : (
-              <span>Brak danych</span>
-            )}
-          </div>
-        </div>
+        {
```

Comment:
I know that page with favourites schools doesn't work really well, but I don't understand what is the purpose of commenting this part. If we want to hide this part of application, then we should probably hide the element in nav.

Reply:
It is only temporary. I removed fetching data from REST API, but still I'd like to use the same layout when we add GraphQL here. It is important but not crucial for the app to work, so imho we can add it in the next PR.

Reply:
ok!